build_tools/github_actions/ci_build_cmake_code_coverage.sh

```diff
@@ -84,12 +84,12 @@ lcov --directory "$STABLEHLO_BUILD_DIR"  \
 # Capture code coverage output
 if [[ $GENERATE_HTML == true ]]; then
   exec 5>&1
-  CCOV_OUT=$(genhtml $LCOV_DATA \
```

Comment:
this removes setting the variable?
Is that needed

Reply:
I don't believe so, the variable is dead.